Reference_Manual.md

```diff
@@ -1217,25 +1217,24 @@ One of the offers of Hazelcast is the TLS/SSL protocol which you can use to esta
 
 ### 6.1.1. TLS/SSL for Hazelcast Members
 
-Hazelcast allows you to encrypt socket level communication between Hazelcast members and between Hazelcast clients and members, for end to end encryption. To use it, see the [TLS/SSL for Hazelcast Members section](http://docs.hazelcast.org/docs/latest/manual/html-single/index.html#tls-ssl-for-hazelcast-members) in the Hazelcast IMDG Reference Manual.
+Hazelcast allows you to encrypt socket level communication between Hazelcast members and between Hazelcast clients and members, for end to end encryption. To use it, see the [TLS/SSL for Hazelcast Members section](https://docs.hazelcast.com/imdg/latest/security/tls-ssl.html#tlsssl-for-hazelcast-members) in the Hazelcast IMDG Reference Manual.
 
 ### 6.1.2. TLS/SSL for Hazelcast C++ Client
 
 The Hazelcast C++ client uses [Boost Asio](https://www.boost.org/doc/libs/1_75_0/doc/html/boost_asio.html) library for networking and secure communication. 
 
-To use TLS/SSL with your Hazelcast C++ client, you should build the library with OpenSSL feature turned on. By default, this feature is not turned on. Here are different ways to install the library with SSL support:
+To use TLS/SSL with your Hazelcast C++ client, you should build the library with OpenSSL feature turned on. By default, this feature is not turned off. Here are the different ways to install the library with SSL support:
```

Comment:
... By default, this feature is turned off.

Reply:
fixed.